src/main/java/online/partyrun/partyrunmatchingservice/domain/waiting/root/RunningDistance.java

```diff
@@ -0,0 +1,29 @@
+package online.partyrun.partyrunmatchingservice.domain.waiting.root;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import online.partyrun.partyrunmatchingservice.domain.waiting.exception.NotAllowMeterException;
+
+import java.util.Arrays;
+
+@Getter
+@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
+@RequiredArgsConstructor
+public enum RunningDistance {
+    M1000(1000),
+    M3000(3000),
+    M5000(5000),
+    M10000(10000);
+
+    int meter;
+
+    public static RunningDistance getByMeter(int meter) {
```

Comment:
```suggestion
    public static RunningDistance findBy(int meter) {
```
위처럼 메서드명 가져가는건 어떨까요?

Reply:
반영했습니다